CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ option(WarpX_LIB           "Build WarpX as a library"                   OFF)
 option(WarpX_MPI           "Multi-node support (message-passing)"       ON)
 option(WarpX_SIMD          "CPU SIMD Acceleration"                      OFF)
 option(WarpX_OPENPMD       "openPMD I/O (HDF5, ADIOS)"                  ON)
+option(WarpX_FASTMATH      "Enable fast-math optimizations"             OFF)  # TODO: set to ON by default!
```

Comment:
The `TODO` comment here may be outdated based on our discussion this morning, but up to you whether you want to keep it as is, change it, or remove it.